config/environments/production.rb

```diff
@@ -87,16 +87,16 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  # Paperclip Amazon S3 hosting
-  config.paperclip_defaults = {
-    storage: :s3,
-    s3_credentials: {
-      bucket: ENV.fetch('S3_BUCKET_NAME', ''),
-      access_key_id: ENV.fetch('AWS_ACCESS_KEY_ID', ''),
-      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY', '')
-    },
-    s3_protocol: :https
-  }
+  # # Paperclip Amazon S3 hosting
```

Comment:
We can remove this, version history will save us if it gets to having to reintroduce it! (I'm sure it won't :))